### DISCUSSION TITLE
A New Year in New to GitHub 🤓  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#44133] [New to GitHub]
Happy 2023 to all our members! 2022 was such a delight with all of you and I can’t wait to see what we accomplish together this year 🎆. 

🌟**Community Superstars**: A round of applause for these community citizens 👏 
* @mariomarinato: For asking a [really interesting question](https://github.com/orgs/community/discussions/43320) that may be relevant to those migrating over to GitHub from another tracking tool ✈️. 
* @diva-21: Created an AWESOME [answer](https://github.com/orgs/community/discussions/43216) with a multitude of ways to get involved in GitHub. If you’re new to GitHub and looking to get involved more in open source, this comment will get you started 🔥. 
* @kalialor: Self-solver 🕵️ on why their contributions weren’t showing up. Way to go!  [Get the solution](https://github.com/orgs/community/discussions/43529). 

📚 **Rousing Reads**: GitHub blog goodies 🍭
* **How we use GitHub to be more productive, collaborative, and secure**: @mph4 shares a [look](https://github.blog/2022-12-20-how-we-use-github-to-be-more-productive-collaborative-and-secure/)  at how we use GitHub to be more productive, build collaboratively, and shift security left.
* **What the duck** 🦆: @mishmanners [explains](https://github.blog/2022-12-23-whats-with-all-the-ducks/) what is with all the ducks in software development. 
* **GitHub’s top 10 blog posts of 2022**: [Read](https://github.blog/2022-12-29-githubs-top-10-blog-posts-of-2022/) the top posts of the past year from the blog. 

🆒 **Cool stuff**™️ happening around GitHub and the Community
* **GitHub Discussions just got better with Category Forms**: Category Forms allow maintainers to create templates for their GitHub Discussions, which means that users can start new discussions with all the necessary information already included 💫. [Get the details](https://github.blog/2023-01-09-github-discussions-just-got-better-with-category-forms/) from @abhinavrajesh. 
* **[Private Beta] Roadmap Feedback Wanted**:  @rileybroughten and team want to know if you’ve encountered any issues or have suggestions on the Roadmap feature for Projects 🛣️. Learn how to join the waitlist and share feedback [here](https://github.com/orgs/community/discussions/39766).
* **Celebrate all we achieved in 2022**: Our [2022 Year in Review](https://github.com/orgs/community/discussions/42328) graphic went live right before the end of year 🚀. 

Got a coding language or GitHub product you hope to learn more about in 2023? I want to hear in the comments below.

**Comment by diva-21** [comment]
Thank You
